vertx-core/src/main/java/io/vertx/core/impl/HostnameResolver.java

```diff
@@ -45,35 +45,26 @@ public class HostnameResolver implements AddressResolver {
 
   private static final Logger log = LoggerFactory.getLogger(HostnameResolver.class);
 
-  private static Pattern resolvOption(String regex) {
-    return Pattern.compile("^[ \\t\\f]*options[^\n]+" + regex + "(?=$|\\s)", Pattern.MULTILINE);
-  }
-
-  private static final Pattern NDOTS_OPTIONS_PATTERN = resolvOption("ndots:[ \\t\\f]*(\\d)+");
-  private static final Pattern ROTATE_OPTIONS_PATTERN = resolvOption("rotate");
+  private static final Pattern WHITESPACE_PATTERN = Pattern.compile("\\s+");
```

Comment:
I'm curious if the following would avoid the apparently heavy initialization of what I suspect is the usage of `\s`
```suggestion
  private static final Pattern WHITESPACE_PATTERN = Pattern.compile("[ \\t]+");
```

Asking because I suspect `resolv.conf` would support only traditional spaces and tabs.

Now maybe it's not worth it.

Reply:
It's certainly worth a try. I'll be out for a few days, so I can try it when I get back if you haven't already

Reply:
I have no idea which app you used to generate this so I think it can wait for a few days.

Reply:
I just used the rest-json Quarkus quickstart on a Linux machine

Reply:
Actually this regexp is not used anymore, AFAICS.

Reply:
Updated